packages/jasmine-enzyme/src/index.js

```diff
@@ -29,3 +29,13 @@ export default function jasmineEnzyme() : void {
     });
   });
 }
+
+// Also expose enzymeMatchers directly so that the matchers can be added on a per-spec basis
+// instead of globally on the jasmine object. This also supports older versions of jasmine where
+// jasmine.addMatchers isn't defined and matchers must be added to the spec in a beforeEach().
```

Comment:
Yeah. I don't think it's a bad move. I'm fine leaving it undocumented though. Shouldn't be a major use-case.

Also, just a shout-out. You could likely convert to jest with minimal effort and maybe use a few codemods.

I'll merge these tomorrow and put out a release.